src/include/job.h

```diff
@@ -1142,26 +1142,23 @@ extern int   do_tolerate_node_failures(job *);
  */
 #ifdef PBS_MOM
 
-extern job  *job_recov_fs(char *);
-extern int   job_save_fs(job *);
+extern job *job_recov_fs(char *);
+extern int job_save_fs(job *);
 
-#define job_save job_save_fs
+#define job_save  job_save_fs
 #define job_recov job_recov_fs
 
 #else
 
-extern job  *job_recov_db(char *, job *pjob);
-extern int  job_save_db(job *);
+extern job *job_recov_db(char *, job *pjob);
+extern int job_save_db(job *);
 
-#define job_save job_save_db
+#define job_save  job_save_db
 #define job_recov job_recov_db
 
-extern char *get_job_credid(char *jobid);
-
+extern char *get_job_credid(char *);
```

Comment:
why are we adding this now?

Reply:
no, I just removed `jobid` word from the argument and an extra blank line.